libs/leaflet-distance-marker.js

```diff
@@ -206,7 +206,7 @@ L.DistanceMarkers = L.LayerGroup.extend({
 									iconSize: options.arrowSize,
 								}),
 								// NB the following option is added by "leaflet-rotate"
-								rotation: angle * L.DomUtil.RAD_TO_DEG,
+								rotation: angle,
```

Comment:
> // NB the following option is added by "leaflet-rotate"
> rotation: angle * L.DomUtil.RAD_TO_DEG,

Before I get into the code, did you notice the comment above? (that is: are you including this library as well? [Raruto/leaflet-rotate](https://github.com/raruto/leaflet-rotate))

Reply:
Hi @Raruto. 
Sorry for long time.
In this moment don't remember if i put the library ( i would want say yes but i'm not sure), 
I will open the project and inspect it a second time.

Reply:
I've visited some of your demo about leaflet-elevation and the arrows don't show the correct direction way.
For example: [leaflet-elevation/i18n-strings](https://raruto.github.io/leaflet-elevation/examples/leaflet-elevation_i18n-strings.html) or
[leaflet-elevation/dynamic-runner](https://raruto.github.io/leaflet-elevation/examples/leaflet-elevation_dynamic-runner.html. the arrows point incorrectly) or so I think.
With this change, the arrows point the correct direction in every moment.
[RouteMap for University of Valencia](https://www.uv.es/fatwirepub/Satellite?pagename=UV/Page/WSRouteMap&c=Page&idRouteMap=1286264935342&locale=es_ES&output=html)

Answering you about the library [Raruto/leaflet-rotate](https://github.com/raruto/leaflet-rotate). No, I tried to use in my first attempt but I didn't need rotate all map. Anyway, the demos of leaflet-elevation don't use the library leaflet-rotate and the arrows point to the wrong direction.  Maybe I'm doing something wrong?

Reply:
@makenshikuro just to know, arrows are shown correctly if instead you set `preferCanvas: true`?

Ref: https://leafletjs.com/reference.html#map-prefercanvas